docs/lib/datastore/fragments/native_common/setup-auth-rules.md

```diff
@@ -0,0 +1,122 @@
+Amplify gives you the ability to limit which individuals or groups should have access to create, read, update, or delete data on your various schema types by specifying an `@auth` directive.
+
+The following is a high level overview of the various authorization scenarios we support in the Amplify libraries.  Each scenario has a variety of options you can tune to fit the needs of your application.
+
+* [**Owner Based Authorization**](#owner-based-authorization): Uses Cognito User Pools as an auth provider to add ownership and access rules to instances of your model
+* [**Static Group Authorization**](#static-group-authorization): Uses Cognito User Pools as an auth provider with groups to apply access rules to instances of your model
+* [**Owner and Static Group Combined**](#owner-and-static-group-combined): Uses a combination of both *Owner Based Authorization* and *Static Group Authorization* to control ownership and access.
+* [**Public Authorization**](#public-authorization): Use API Key or IAM to allow public access to your model instances
+* [**Private Authorization**](#private-authorization): Use IAM or Cognito User Pools to allow private access to your model instances
+* [**Owner based Authorization with OIDC provider**](#owner-based-authorization-with-oidc-provider): Use a 3rd party OIDC Provider to achieve *Owner based authorization*
+* [**Static Group Authorization with OIDC provider**](#static-group-authorization-with-oidc-provider): Use a 3rd party OIDC Provider to achieve *Static group authorization* using a custom `groupClaim`.
+
+## Commonly used @auth rule patterns
+
+### Owner Based Authorization
+
+The following are commonly used patterns for owner based authorization.  For more information on how to tune these examples, please see the [CLI documentation on owner based authorization](~/cli/graphql-transformer/auth.md#owner-authorization).
+
+* Create/update/delete mutations are private to the owner.
+```graphql
+type YourModel @model @auth(rules: [{ allow: owner }]) {
+  ...
+}
```

Comment:
all the `type YourModel @model { }` stuff is super repetitive and doesnt add anything, suggest doing 1 instance of it and then the rest just show `@auth(whatever)`

Reply:
I was thinking to this but there's also field level auth, so i wanted to make it clear that we are using `@auth` in conjunction with `@model`.  Thoughts?

Reply:
we could mention that these rules could be applied either at model level or at row level?

Reply:
Unfortunately, row level (field level) is not fully supported.  There are known issues with regards to using field level auth... :(  I don't think that this is a huge blocking to moving forward, so, appreciate the suggestion, but I think it's fine to leave it as is for now.